en_ID/navigation.md

```diff
@@ -42,13 +42,23 @@
 
 [Social]()
 
+<<<<<<< HEAD
```

Comment:
I think this is coming from git conflict and not necessary.

Reply:
oww, sorry. I fixed that.